include/iio/iio.h

```diff
@@ -876,6 +876,12 @@ __api __check_ret __pure const char * iio_channel_get_id(const struct iio_channe
  * <b>NOTE:</b> if the channel has no name, NULL is returned. */
 __api __check_ret __pure const char * iio_channel_get_name(const struct iio_channel *chn);
 
+/** @brief Retrieve the channel name (e.g. <b><i>anglY</i></b>)
```

Comment:
copy/paste? Shouldn't this say `Retrieve the channel label`?

Reply:
yes, it should. And yes, it was a copy/paste side effect :)